src/openkit-shared/Protocol/Beacon.cs

```diff
@@ -84,7 +85,7 @@ public class Beacon
         private const string BEACON_KEY_WEBREQUEST_BYTES_RECEIVED = "br";
 
         // max name length
-        private const int MAX_NAME_LEN = 250;
+        internal const int MAX_NAME_LEN = 250;
```

Comment:
as you touched the const here. you might consider changing the name to comply with the C# naming conventions of constants. this also applies to other constants defined in Beacon.cs